### ISSUE TITLE
Multiline text field: text colour is wrong

### ISSUE THREAD

**Issue by rachel-fenichel** [rachel-fenichel/blockly-samples#2517]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Component

`field-multiline-text`

### Description

Default text colour in the test playground is white instead of black.

### Reproduction steps

Link against blockly v12.0.0-beta.5 and open the plugin test page.

### Stack trace

```shell

```

### Screenshots

![Image](https://github.com/user-attachments/assets/7fa920b4-7528-47f1-9868-d0560d6c884b)
